src/main/tray/tray.js

```diff
@@ -45,16 +45,16 @@ export function refreshTrayImages(trayIconTheme) {
         const theme = trayIconTheme;
         try {
             trayImages = {
-                normal: nativeImage.createFromPath(path.resolve(assetsDir, 'linux', theme, 'MenuIcon.png')),
-                unread: nativeImage.createFromPath(path.resolve(assetsDir, 'linux', theme, 'MenuIconUnread.png')),
-                mention: nativeImage.createFromPath(path.resolve(assetsDir, 'linux', theme, 'MenuIconMention.png')),
+                normal: nativeImage.createFromPath(path.resolve(assetsDir, 'linux', 'top_bar_dark_16.png')),
```

Comment:
The changes in this file reflect the renamed PNG files (which I also moved from the subfolders to the parent folder because they have distinct names, now).